package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "quill-todo",
+  "version": "1.0.0",
+  "description": "A better todo app",
+  "main": "tasks-fe\\build\\index.html",
+  "scripts": {
+    "init-be-win": "workon quill && pip install -r requirements.txt && pg_ctl start && chcp 1252 && psql -d postgres -a -f setup.sql && py manage.py makemigrations && py manage.py migrate",
+    "init-be-mac": "workon quill && pip install -r requirements.txt && psql -d postgres -a -f setup.sql && python3 manage.py makemigrations && python3 manage.py migrate",
```

Comment:
since everyone's virtual environment is named something different, `workon quill` won't properly start for everyone. I'm not sure if this is mentioned in the readme but if not then we should make a note to start the virtual env each time before starting to work on it (after it's already been created the first time). aside from that, the script for mac is correct.

Reply:
Sweet! Yep, it's mentioned in contributing